frontend/src/components/Mobile/InputField.tsx

```diff
@@ -55,44 +63,38 @@ const StyledTitle = styled.div`
 `
 
 type InputFieldProps = {
-  title?: string
+  title: string
   placeholder: string
   value?: string
   setValue?: Dispatch<SetStateAction<string>> | ((input: string) => void)
   textArea?: boolean
-  hasError?: boolean
-  onChange?: () => void
+  errors: { [x: string]: any }
```

Comment:
the type for this might be `FieldError` see [link](https://react-hook-form.com/ts#FieldError)